src/lib/viewers/ConfigViewer.ts

```diff
@@ -1,20 +1,28 @@
 import {Display} from '../Display';
 import {ConfigModel, OutputFormat} from '../models/ConfigModel';
+import {toComment} from '../utils/StylingUtil';
 
 
 export interface ConfigViewer {
 	view(configModel: ConfigModel): void;
 }
 
-export class ConfigRawYamlViewer implements ConfigViewer {
+export class ConfigStyledYamlViewer implements ConfigViewer {
 	private readonly display: Display;
 
 	public constructor(display: Display) {
 		this.display = display;
 	}
 
 	public view(configModel: ConfigModel): void {
+		const styledYaml = configModel.toFormattedOutput(OutputFormat.YAML)
+			// We're seeking `#`, the YAML comment character, and comment-styling that character and the rest of the line.
+			// Known Bug: We don't distinguish between `#` characters inside and outside of strings or the like. In the
+			//            event that a `#` is part of a string, it will still be perceived by this line as part of a
+			//            comment and formatted as such. For now, we're accepting this, because it's an unlikely scenario
+			//            to begin with, and because it's purely a cosmetic UI issue.
+			.replaceAll(/#.+/gm, s => toComment(s));
```

Comment:
In the next PR, you'll have all the comments from the overview and fieldDescriptions... so you'll know exactly what the comments are.

I also believe your implementation will change quite a bit. But for now this is good enough. The only thing I would ask is to have the "toComment" function be called something else... like "makeGrey".

Reply:
Yeah, `makeGrey` is probably a better name. Stand by.